tables/basic-table/step6.md

```diff
@@ -0,0 +1,44 @@
+In the next few steps, we'll dive into incorporating the Pagination component so users can better control their view of the data.
+
+## Task
+
+1) Import Pagination component from react-core and render it just above the Table component.
+
+<strong>Note: </strong> Group the Table and Pagination together withing a single fragment `<>` container.
+
+<pre class="file" data-target="clipboard">
+import { Pagination } from '@patternfly/react-core';
+</pre>
+
+2) Set the itemCount prop of Pagination to the total number of rows in your dataset
+
+<pre class="file" data-target="clipboard">
+itemCount={defaultRows.length}
```

Comment:
I don't see a step to add the pagination component, yet? (Although, I see it in the final step below.) It would be helpful to show exactly where in the code that should be at this step.